spec/spec_helper.rb

```diff
@@ -0,0 +1,100 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
```

Comment:
please take a look at how we setup rspec in the course documentation for "Rails AUT". You are missing some configuration

Reply:
I think this is fine.  I added the shoulda-matchers to the 'rails-helper', is that what you are referring to?